.buildkite/scripts/tooling.sh

```diff
@@ -74,3 +80,112 @@ running_on_buildkite() {
     fi
     return 1
 }
+
+create_elastic_package_profile() {
+    local name="$1"
+    elastic-package profiles create "${name}"
+    elastic-package profiles use "${name}"
+}
+
+prepare_serverless_stack() {
+    echo "--- Prepare serverless stack"
+    local stack_version=${STACK_VERSION:-""}
+
+    local args="-v"
+    if [ -n "${stack_version}" ]; then
+        args="${args} --version ${stack_version}"
+    fi
+
+    # Currently, if STACK_VERSION is not defined, for serverless it will be
+    # used as Elastic stack version (for agents) the default version in elastic-package
```

Comment:
Nit. Move this comment closer to `elastic-package stack up`?

Reply:
Done.